lib/libLog.lua

```diff
@@ -0,0 +1,378 @@
+--[[
```

Comment:
dtutils came from existing code pulled from the scripts.  libLog was a new thing and I didn't want to get the two mixed up.  Moving it to dtutil.log is fine.  I just have to make sure that I don't get in a circular require loop, requiring libraries from each other.  That gets nasty real fast ;).